ci/python-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Upload Python Package
+
+on: [release]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      if: github.event.action == 'created'
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      if: github.event.action == 'created'
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      if: github.event.action == 'created'
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
```

Comment:
And I think here ideally this could use PEP517 builders so that it works regardless of whether someone's using setuptools or not. I.e. `pep517.build .`

Reply:
I think it's best to stick to the current command that the official documentation on [packaging python projects](https://packaging.python.org/tutorials/packaging-projects/) uses.

Reply:
Yep makes sense.